### PR TITLE
Fixed #24423 -- Combine i18n template tag tests

### PR DESCRIPTION
**Pull request by s0b0lev**
1. Move all template tag (trans and blocktrans) tests from **i18n/test.py** to **template_tests/test_i18n_tags.py**
2. Split trans and blocktrans template tags from **template_tests/syntax_tests/test_i18n.py** into separate files:  **template_tests/syntax_tests/test_trans.py**, **template_tests/syntax_tests/test_blocktrans.py**.

### CODE REVIEW

**Comment by timgraham** [comment]
buildbot, add to whitelist.

File: tests/template_tests/test_i18n_tags.py [hunk 1]
```diff
@@ -0,0 +1,360 @@
+# coding: utf-8
```
Comment (timgraham) [inline tests/template_tests/test_i18n_tags.py]: We standardize on `# -*- coding: utf-8 -*-`.

File: tests/template_tests/test_i18n_tags.py [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+
+    @override_settings(LOCALE_PATHS=[os.path.join(here, 'other', 'locale')])
+    def test_bad_placeholder_1(self):
+        """
+        Error in translation file should not crash template rendering
+        (%(person)s is translated as %(personne)s in fr.po)
+        Refs #16516.
+        """
+        with translation.override('fr'):
+            t = Template('{% load i18n %}{% blocktrans %}My name is {{ person }}.{% endblocktrans %}')
```
Comment (timgraham) [inline tests/template_tests/test_i18n_tags.py]: Is there a reason why there are still tests for {% trans %} and {% blocktrans %} in here as opposed to putting them in the other tag-specific test files?

Comment (s0b0lev) [inline tests/template_tests/test_i18n_tags.py]: Ticket has two parts:
1) Move the tag-related tests from **i18n** into **template_tests**
2) Split **template_tests.syntax_tests.test_i18n** into separate files for each tag: **test_trans.py** and  **test_blocktrans.py**.

First part is about putting tests from **i18n/test.py** to template_tests (this tests not about syntax, if I have understood properly). 
Second part is about syntax specific tests, here I split trans and block trans tags to **template_tests/syntax_tests/test_trans.py** and **test_template_tests/syntax_tests/test_blocktrans.py**

So thats why I have split tests only from **template_tests.syntax_tests.test_i18n**.
And move all tag-related tests from **i18n/test.py** to **template_tests/test_i18n_tags.py**

If it's wrong structure, I'll create commit with this changes.

Comment (timgraham) [inline tests/template_tests/test_i18n_tags.py]: I don't know know if there's any distinction between a "syntax" tests and a "non-syntax" test. I think the fact that the the tags were split in the first place between template_tests and i18n is just accidental. @claudep, any input?

**Comment by s0b0lev** [comment]
@timgraham is that problems [BUILD-8832](http://djangoci.com/job/pull-requests-trusty/database=mysql,label=trusty-pr,python=python2.7/8832/console) with CI?

**Comment by timgraham** [comment]
Looks like a transient build failure, nothing to worry about.

**Comment by s0b0lev** [comment]
@timgraham , so may I just put all tests for {% trans %} and {% blocktrans %} in tag-specific files?

**Comment by timgraham** [comment]
I think so.

**Comment by s0b0lev** [comment]
Done

**Comment by timgraham** [comment]
I rebased this after some recent changes and squash commits but I think the organization could still be improved a bit. There's `test_i18n.py` and `test_i18n_tags.py` which are similar. How about creating a `syntax_tests/i18n` directory and creating a separate test file for each i18n tag?

**Comment by s0b0lev** [comment]
Ok, I will do it.

**Comment by s0b0lev** [comment]
@timgraham I wanna to clear understand what to do)
1. I create dir syntax_tests/i18n
2. Create two files test_i18n_trans.py, test_i18n_blacktrans.py
3. Move tests only from this files: " _There's test_i18n.py and test_i18n_tags.py which are similar._? Or move tests from all files with tans/blocktrans related tests?

**Comment by timgraham** [comment]
In syntax_tests/i18n: test_trans.py, test_blocktrans.py, test_get_language_info.py, etc. (a separate file for each i18n template tag). No more miscellaneous files like test_i18n_tags.py and test_i18n.py unless you find something that doesn't fit into a specific file.

**Comment by s0b0lev** [comment]
Got it, thank you.

**Pull request closed without merging**